src/DynamicMvvm.Uno.WinUI/DynamicMvvm.Uno.WinUI.csproj

```diff
@@ -0,0 +1,55 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+	<PropertyGroup>
+		<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
+		<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;net6.0-ios;net6.0-maccatalyst;net6.0-macos</TargetFrameworks>
```

Comment:
I don't think this condition is ever met because the pipeline only builds on windows...